packages/ripple-ui-components/scripts/generate-component/stories_template.js

```diff
@@ -6,17 +6,17 @@ import {
   withKnobs
```

Comment:
As storybook has been upgraded to 5, a lot changes happened. This is outdated. Please update it according to latest changes in storybook.